src/app/pages/quickMaps/quickMaps.component.ts

```diff
@@ -0,0 +1,41 @@
+
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, NavigationEnd, Router } from '@angular/router';
+import { QuickMapsRouteData } from './quickMaps-routing.module';
+import { QuickMapsService } from './quickMaps.service';
+import { Subscription } from 'rxjs';
+
+
+@Component({
+  selector: 'app-quickmaps',
+  templateUrl: './quickMaps.component.html',
+  styleUrls: ['./quickMaps.component.scss'],
+})
+export class QuickMapsComponent implements OnInit {
+
+  public showHeader = true;
+
+  constructor(
+    router: Router,
+    private activatedRoute: ActivatedRoute,
+    public quickMapsService: QuickMapsService,
+  ) {
+    router.events.subscribe((event) => {
+      if (event instanceof NavigationEnd) {
+        let subs: Subscription;
+        // eslint-disable-next-line prefer-const
+        subs = this.activatedRoute.firstChild.data.subscribe((data: QuickMapsRouteData) => {
```

Comment:
can this not be const subs and remove the eslint disable?

Reply:
It didn't like that at runtime.  It was saying that the variable hadn't been initialized.